### ISSUE TITLE
Make show() / dismiss() public

### ISSUE THREAD

**Issue by mhoberhausen** [iammert/MaterialIntroView#41]
Would be nice to programmatically dismiss the MaterialIntroView so could you make the dismiss function in MaterialIntroView.java public? This is already added to this PR #28 which includes the rectangle shapes (also useful :) )

**Comment by starkej2** [comment]
I agree! This would be useful for me as well. I'm trying to create some tutorials that don't dismiss immediately when touched, but instead after a user takes a certain action.

**Comment by jlitjens** [comment]
This would be fantastic!
